src/Core/Varargs/IVarargsFormatParser.cs

```diff
@@ -21,7 +21,7 @@
 using System.Collections.Generic;
 using Reko.Core.Types;
 
-namespace Reko.Core
+namespace Reko.Core.Varargs
 {
```

Comment:
I suggest calling this namespace `Reko.Core.Analysis`, so that other shared analysis classes can go here.